Client/States/GameplayState.cpp

```diff
@@ -99,13 +104,66 @@ void GameplayState::Update(float dt) {
 void GameplayState::ReadNetworkFunctions() {
     while (!networkData->incomingFunctions.IsEmpty()) {
         FunctionPacket packet = networkData->incomingFunctions.Pop();
-        if (packet.functionId == Replicated::AssignPlayer) {
-            DataHandler handler(&packet.data);
-            auto networkId = handler.Unpack<int>();
-            AssignPlayer(networkId);
+        DataHandler handler(&packet.data);
```

Comment:
I think handling the camera movement across the network is fine, maybe check this with joe